dataikuapi/dss/recipe.py

```diff
@@ -1340,6 +1340,27 @@ class EvaluationRecipeCreator(DSSRecipeCreator):
         builder.with_output_evaluation_store(evaluation_store_id)
 
         new_recipe = builder.build()
+
+        # Access the settings
+
+        er_settings = new_recipe.get_settings()
+        json_payload = er_settings.get_json_payload()
```

Comment:
This is a legacy way of accessing the payload. The recommended way is:

```
payload = er_settings.obj_payload
```

Notably, "json" is an implementation detail, what matters is that it is a dict